Task_2_1_1/src/main/java/r/masud/ParallelStreamBenchmark.java

```diff
@@ -0,0 +1,47 @@
+package r.masud;
+
+import org.openjdk.jmh.annotations.*;
+import java.util.Arrays;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+@BenchmarkMode(Mode.AverageTime)
```

Comment:
And also 2.1.1 should not be in PR by 2.2.1

You should keep labs in separate branches and branch them all from main, not from each other

Reply:
yes, but I had a minor problem so I had to brunch them from diffrent, root. I'II fix it with the next task